lib/compiler.js

```diff
@@ -313,18 +310,25 @@ function resolveSourceDir(rootDir, sourceDir) {
     var modulePaths = Module.globalPaths
       .concat(Module._nodeModulePaths(rootDir));
 
-    srcDir = modulePaths
+    fullPath = modulePaths
       .map(function(candidateDir) {
-        return path.join(candidateDir, sourceDir);
+        return path.join(candidateDir, relativePath);
       })
       .filter(function(candidate) {
         return fs.existsSync(candidate);
       })
       [0];
-    if (srcDir)
-      return srcDir;
+    if (fullPath)
+      return fullPath;
+  } else {
+    // Handle relative path, e.g. `{appRootDir}/common/models`
+    try {
+      fullPath = require.resolve(fullPath);
+      return fullPath;
+    } catch (err) {
+      debug ('Skipping %s - %s', fullPath, err);
+    }
   }
-
```

Comment:
Nitpick: please keep the empty line as a visual delimiter of different code blocks.